test/test_find_studies.py

```diff
@@ -0,0 +1,41 @@
+import xml.etree.ElementTree as ET
+
+import pytest
+
+from brdnet import find_studies
+
+
+group1 = {'samples': ['a']}
+group2 = {'samples': ['b']}
```

Comment:
Doesn't look like `group2` is being used anywhere?

Reply:
Good catch, it was being used in a previous test that I ended up removing